SimTracker/TrackAssociatorProducers/plugins/QuickTrackAssociatorByHitsImpl.h

```diff
@@ -5,6 +5,8 @@
 
 #include "SimDataFormats/Associations/interface/TrackToTrackingParticleAssociator.h"
 
+#include "SimTracker/TrackerHitAssociation/interface/ClusterTPAssociationProducer.h"
```

Comment:
I believe it is bad policy to ever allow other code to read the header file for a producer. I strongly suggest moving the typedef somewhere else.